addon/kubeflow-operator/manifest.yaml

```diff
@@ -482,7 +477,9 @@ spec:
       - istio
       parameters:
       - name: admin
-        value: johnDoe@acme.com
+        value: ""
```

Comment:
Why do we need to set this `admin` to empty value?

Reply:
It's not really needed, it just makes more sense to have no admin rather than `johnDoe@acme.com`.
Actually, in our plugin, perhaps it would make sense to make this equal to the cluster owner (not sure what is the difference with admin role, will try it).